vendor/k8s.io/kubernetes/pkg/kubectl/cmd/testdata/edit/testcase-apply-edit-last-applied-syntax-error/2.edited

```diff
@@ -1,7 +1,7 @@
 # Please edit the 'last-applied-configuration' annotations below.
 # Lines beginning with a '#' will be ignored, and an empty file will abort the edit.
 #
-# The edited file had a syntax error: error converting YAML to JSON: yaml: line 12: could not find expected ':'
+# The edited file had a syntax error: error converting YAML to JSON: yaml: line 13: could not find expected ':'
```

Comment:
these files are UPSTREAM changed. Why are they there in the first place?

Reply:
In the last two PR's, 3.10 and 3.11, @deads2k had me keep the required unit test changes separate so they could be easily checked. I just continued that pattern to be consistent but I can see the reasoning of just combining now if you prefer.

Reply:
yes, extra commit please.